chore: update README files (#10451)
emqx/emqx: apps/emqx_conf/README.md

# Configuration Management

This application provides configuration management capabilities for EMQX.

> This includes, during compilation:
> - Read all configuration schemas and generate the following files:

Reviewer: Nit, simpler language:
```suggestion
At compile time it reads all configuration schemas and generates the following files:
```
